journey-map/public/index.mjs

```diff
@@ -67,6 +67,8 @@ const stateSearch = document.getElementById('state-search');
 const journeyDesc = document.getElementById('journeyDesc');
 const journeyName = document.getElementById('journeyName');
 
+document.getElementById('header-title').href = window.location.href.split('?')[0];
```

Comment:
I'd prefer this to match the mechanism used in our other links - we can leave the href fixed, but override the click behaviour.

Something like:
```
link.onclick = async (e) => {
  e.preventDefault();
  await switchJourney(DEFAULT_JOURNEY_TYPE);
};
```

I've also been trying to put all the selectors in the block above, and any 'initialisation logic' in the `initialize` method at the bottom - that makes it easier to keep track of all the bits and pieces.

Reply:
@Joe-Edwards-GDS thanks, done :)